DQM/BeamMonitor/plugins/AlcaBeamMonitor.cc

```diff
@@ -174,6 +173,7 @@ void AlcaBeamMonitor::bookHistograms(DQMStore::IBooker& ibooker, edm::Run const&
     }
   }
   ibooker.setCurrentFolder(monitorName_ + "Service");
+  auto scope = ibooker.setScope(MonitorElementData::Scope::LUMI);
```

Comment:
The better C++ way to do such behavior is to use RAII (Resource Acquisition Is Initialization - resource release is destruction).
```
{
     DQMScopeContext scope{ ibooker, MonitorElementData::Scope::LUMI };
     theValuesContainer_ = ...
}
```
at the end of the block scope, the `scope` object is destroyed and it resets the ibooker scope back to what it started at.

Reply:
This was not exactly how I expected this API to be used, but it turned out to be a common pattern for migrating the existing code. I might add a helper like this.